includes/compat/class-wc-stripe-email-failed-preorder-authentication.php

```diff
@@ -0,0 +1,70 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Failed Renewal/Pre-Order Authentication Notification
+ *
+ * @extends WC_Stripe_Email_Failed_Authentication
+ */
+class WC_Stripe_Email_Failed_Preorder_Authentication extends WC_Stripe_Email_Failed_Authentication {
+	/**
+	 * Constructor.
+	 *
+	 * @param WC_Email[] $email_classes All existing instances of WooCommerce emails.
+	 */
+	public function __construct( $email_classes = array() ) {
+		$this->id             = 'failed_preorder_sca_authentication';
+		$this->title          = __( 'Pre-order Payment Action Needed', 'wc-pre-orders' );
```

Comment:
Is this the correct text domain?

Reply:
Teeechnically it would be correct, since those strings are coming from pre-orders and they'd probably be translated, but no, it's not. While cleaning up text domains I was looking to replace `woocommerce-pre-orders`, not realising that they are using the `wc-` prefix. Either way, thank you for noticing!

Fixed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/991/commits/bfb78cc76c39d3ecb945ed4e49d493fa442b9790